frontend/next.config.js

```
@@ -4,6 +4,22 @@ const nextConfig = {
     output:'standalone',
     webpack: (config) => {
       config.resolve.alias['@'] = path.resolve(__dirname);
+
+      const fileLoaderRule = config.module.rules.find(
+        (rule) => rule.test && rule.test.test?.('.svg')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)